build-rules/travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+sudo: required
+services:
+  - docker
+matrix:
+  include:
+  - go: 1.10.3
```

Comment:
Can we update this to 1.11? I believe some repos like external-provisioner have already been updated

Reply:
Sure.

That only external-provisioner was updated shows that we need a mechanism like this that makes it simpler to ensure consistency.